win32/src/PyUnicode.cpp

```diff
@@ -103,12 +103,17 @@ BOOL PyWinObject_AsString(PyObject *stringObject, char **pResult, BOOL bNoneOK /
     }
     // Convert the string if a WIDE string.
     if (PyUnicode_Check(stringObject)) {
+#if defined(PYPY_VERSION_NUM) && PYPY_VERSION_NUM < 0x07030b00 
+        stringObject = tempObject =
+            PyUnicode_EncodeMBCS(PyUnicode_AS_UNICODE(stringObject), PyUnicode_GET_SIZE(stringObject), NULL);
+#else
         // PyUnicode_EncodeMBCS was removed in Py 3.11.
         PyObject *unicode = PyUnicode_FromWideChar(PyUnicode_AS_UNICODE(stringObject), -1);
         if (unicode == NULL)
             return FALSE;
         stringObject = tempObject = PyUnicode_EncodeCodePage(CP_ACP, unicode, NULL);
```

Comment:
So you know, this was removed in https://github.com/mhammond/pywin32/pull/1860 , you might wanna pull upstream again

Reply:
thanks, see #2